src/header/header.scss

```diff
@@ -264,5 +292,54 @@
         }
       }
     }
+
+    .iui-split-menu {
+      display: inline-flex;
+      justify-content: center;
+      @include iui-focus;
+
+      svg {
+        width: 16px;
+        @include themed {
+          fill: var(--iui-icons-color-actionable);
+        }
+      }
+
+      &.iui-active {
+        svg {
+          fill: var(--iui-icons-color-primary);
+        }
+      }
+
+      svg {
+        border-radius: 50%;
+        padding: $iui-xxs;
+      }
+
+      &:hover,
+      &:focus-visible {
```

Comment:
Three problems here:
1. This selector will be invalid in IE/Safari and break hover styling too (see [one-invalid-pseudo-selector-equals-an-entire-ignored-selector](https://css-tricks.com/one-invalid-pseudo-selector-equals-an-entire-ignored-selector/))
2. Declaring focus-visible styles is the wrong approach. The "proper" way to do it is to just add normal focus styles, and negate non-visible styles like this:
```scss
:focus {
  ...
}
:focus:not(:focus-visible) {
  // undo styles from above
}
```
3. Using background is probably not the best focus style. Outline/border/box-shadow would make more sense.

Reply:
Styling is changed. Not sure if still relevant